File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.cc
#include <iostream>
using namespace std;

int main()
{
	int a, b, c;

	cin >> a;
	cin >> b;
	cin >> c;

	
	if ((a + b + c == 180))
	{
	    if (a == 60 && b == 60 && c == 60)
		    cout << "Equilateral";
		else if (a == b || b == c || c == a)
			cout << "Isosceles";
		else if (a != b != c)
			cout << "Scalene";
	}
	else
		cout << "Error";
}

File: 백준/Bronze/10101. 삼각형 외우기/삼각형 외우기.java
import java.io.*;
import java.util.*;

public class Main {
    public static void main(String args[]) throws IOException {
        Scanner sc = new Scanner(System.in);
        
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();
        
        boolean check = false;
        
        int sum = a + b + c;
        
        if(sum != 180)
        {
            System.out.println("Error");
        }
        else
        {
            if(a == b && b == c)
                System.out.println("Equilateral");
            else if((a==b && b != c) || (a==c && a != b) || (b==c && b != a))
                System.out.println("Isosceles");
            else
                System.out.println("Scalene");
        }
    }
}